src/ensureApiEnabled.ts

```diff
@@ -43,10 +43,23 @@ export async function check(
  * @param apiName The name of the API e.g. `someapi.googleapis.com`.
  */
 export async function enable(projectId: string, apiName: string): Promise<void> {
-  return api.request("POST", `/v1/projects/${projectId}/services/${apiName}:enable`, {
-    auth: true,
-    origin: api.serviceUsageOrigin,
-  });
+  return api
```

Comment:
Please use `try/catch` with `await` here to be consistent with current practices.